src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Models/BankIdLoginApiInitializeRequest.cs

```diff
@@ -9,13 +9,12 @@ internal BankIdLoginApiInitializeRequest()
             
         }
 
-
-        public string PersonalIdentityNumber { get; set; }
+        public string? PersonalIdentityNumber { get; set; }
 
         [Required]
-        public string ReturnUrl { get; set; }
+        public string? ReturnUrl { get; set; }
```

Comment:
The same here, I just curios how runtime behaves with this combination

Reply:
Same thing there, ASP.NET Core will (from my understanding) set it to null if no valye is passed from the client.

Reply:
See details here:
https://github.com/aspnet/AspNetCore/issues/14451#issuecomment-535403894

> As far as I know, modelbinding is not affected in any way by the use of Nullable types in C#. Nullable types help the compiler specify the behavior, but don't offer any runtime guarantee about it nor extend the range of values a type can take.

Basicallt, C#8 nullable are pure compile time helper for the developer. A non nullable string can still be set to null... :)